docs/privacyidea.txt

```diff
@@ -68,6 +68,14 @@ Then you need to add the authentication source 'privacyidea:privacyidea' to
          			 'mobile' => 'mobilePhone'
          			 ),
 
+        /*
+         * You are able to concatenate attributes like the given and surname.
+         * Optional.
+         */
+         'concatenationmap' => array(
+         			'givenname,surname' => 'fullName',
```

Comment:
@Mipronimo  This is all perfect.
But I am wondering, if we (I) have chosen a bad to remember configuration way in the first place.

It felt more intuitive to me to configure

~~~~php
    fullName => givenname,surname
~~~~

since fullName is constructed from these both.

But it is right. In this sense the attributemap would also be the wrong way:

The SAML attribute "samlLoginName" would be filled by the privacyIDEA return value "username":

~~~~php
     "samlLoginName" => "username"
~~~~